RouteStepTests.swift

```diff
@@ -1,6 +1,93 @@
 import XCTest
 @testable import MapboxDirections
 
+class RoadTests: XCTestCase {
+    func testEmpty() {
+        let r = Road(name: "", ref: nil, destination: nil, rotaryName: nil)
+        XCTAssertNil(r.names)
+        XCTAssertNil(r.codes)
+        XCTAssertNil(r.destinations)
+        XCTAssertNil(r.destinationCodes)
+        XCTAssertNil(r.rotaryNames)
+    }
+
+    func testNamesCodes() {
+        var r: Road
+
+        // Name only
+        r = Road(name: "Way Name", ref: nil, destination: nil, rotaryName: nil)
+        XCTAssertEqual(r.names!, [ "Way Name" ])
+        XCTAssertNil(r.codes)
+        r = Road(name: "Way Name 1; Way Name 2", ref: nil, destination: nil, rotaryName: nil)
```

Comment:
Add a test for values delimited by a colon but not a space, since that's the usual OSM tagging convention.